src/main/java/pl/envelo/moovelo/service/event/EventService.java

```diff
@@ -40,4 +50,20 @@ public Event getEventById(Long id) {
         log.info("EventService - getEventById() return {}", eventOptional.get());
         return eventOptional.get();
     }
+
+    public void removeEventById(long id) {
+        log.info("EventService - removeEventById() - id = {}", id);
+        Optional<Event> eventOptional = eventRepository.findById(id);
+        if (eventOptional.isEmpty()) {
+            throw new NoContentException("Event with id = " + id + " doesn't exist!");
+        } else {
+            Event event = eventOptional.get();
+            Location location = event.getEventInfo().getLocation();
+            EventOwner eventOwner = event.getEventOwner();
+            eventRepository.delete(event);
+            locationService.checkIfLocationContainsEvents(location);
+            eventOwnerService.checkIfEventOwnerContainsEvents(eventOwner);
```

Comment:
Czy taki sam mechanizm nie powinien zostać zastosowany także do hashtagów? Powinniśmy usuwać tylko te, które po usunięciu eventu będą miały pustą listę eventów. To nie jest z tego pr ale CascadeType z List<Hashtag> w evencie też będzie do usunięcia